cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceMeta.java

```diff
@@ -140,6 +142,16 @@ public Builder setKeytabURI(String keytabURI) {
       return this;
     }
 
+    public Builder incrementKeytabURIVersion() {
+      // Initialize the keytab URI version to 1 if it is null, otherwise increment the existing version by 1
+      keytabURIVersion = keytabURIVersion == null ? 1 : keytabURIVersion + 1;
```

Comment:
Why initialize to 1 here but then in the handler it gets reset to 0 if the keytab uri is a new one? Shouldn't it be consistent (starts with 0 or starts with 1)

Reply:
because `NamespaceMeta` from pre-5.0 has null in `keytabURIVersion` which is equivalent to version 0 in `NamespaceMeta` created in 5.0. Therefore, the null version is incremented to 1 instead of initialized to 0.